SpatialGDK/Source/SpatialGDK/Private/Tests/SpatialView/SpatialViewUtils.h

```diff
@@ -0,0 +1,168 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "ComponentTestUtils.h"
+#include "SpatialView/ViewDelta.h"
+#include "SpatialView/WorkerView.h"
+
+namespace SpatialGDK
+{
+enum EntityStatus
```

Comment:
nit: `EntityChangeType` maybe? It doesn't seem to be the status of the entity as much as how it is about to change